python/gtsam_py/python/examples/Pose2SLAMExample_g2o.py

```diff
@@ -53,15 +53,16 @@ def vector3(x, y, z):
 assert args.kernel == "none", "Supplied kernel type is not yet implemented"
 
 # Add prior on the pose having index (key) = 0
-priorModel = gtsam.noiseModel_Diagonal.Variances(vector3(1e-6, 1e-6, 1e-8))
-graph.add(gtsam.PriorFactorPose2(0, gtsam.Pose2(), priorModel))
+graphWithPrior = graph
```

Comment:
This script changed significantly and has to be re-ported

Reply:
this one is also virtually the same

Reply:
No